broker/pkg/testing/util/kubernetes.go

```diff
@@ -18,7 +18,6 @@ import (
 	"fmt"
 	"testing"
 	"time"
-
```

Comment:
how did this pass lintesr?

Reply:
The linters seem to have been broken (or with reduced functionality) since about mid-Dec.  I have another PR that tries to get the non-docker working, and I think someone  else is working on updating the linter docker image.

Reply:
This is result of 'fmt.sh' - they should be in sync